src/pricing/cc-pricing-product-consumption.js

```diff
@@ -488,12 +487,10 @@ export class CcPricingProductConsumption extends withResizeObserver(LitElement)
 
   static get styles () {
     return [
-      defaultThemeStyles,
       skeletonStyles,
       // language=CSS
       css`
         :host {
-          background-color: #fff;
```

Comment:
this is debatable but I don't think we need a background for this component. Transparent works just fine and partly fixes the issue we face in the docs website. If we set it to `--cc-colors-bg-default`, I think it looks weird:
![image](https://user-images.githubusercontent.com/100240294/178307734-28497098-a94e-4af3-a9a4-949059a1d48d.png)
vs with no bg defined:
![image](https://user-images.githubusercontent.com/100240294/178307861-8c06f0bb-edae-4174-8a24-600d28b1896a.png)